bibliography-sage.bib

```diff
@@ -3606,7 +3606,7 @@ @article{Marquez-CorbellaMartinez-Moro2012
 }
 
 @article{Martinez-MoroEtAl2013,
-  author       = {Martínez-Moro, E. and Pi\tilde{n}era-Nicolás, A. and Rúa, I. F.},
+  author       = {Martínez-Moro, E. and Piñera-Nicolás, A. and Rúa, I. F.},
```

Comment:
this works

![Screenshot from 2022-02-25 17-34-06](https://user-images.githubusercontent.com/207405/155752100-60d60606-2f8d-42f5-a9e0-d03b50fd41ad.png)